### ISSUE TITLE
looping sounds/sounds don't stop when source/lua tells them to.

### ISSUE THREAD

**Issue by TheVogels** [Mee12345/gmod-infinite-map#103]
Sounds that are playing won't stop when they receive stopsound. If its looping it will play forever until you do stopsound in console. I think this is a bug with your sound detour.

**Comment by Mee12345** [comment]
which sounds specifically?

**Comment by TheVogels** [comment]
I will post a video to show it.

**Comment by TheVogels** [comment]
https://youtu.be/hK2E-sn0ou0?t=136

some audio works right but the ones I mostly show in the video are broken ones.

**Comment by Mee12345** [comment]
hm, what gun addon is causing this issue?

**Comment by TheVogels** [comment]
its also simphys. I will check

**Comment by TheVogels** [comment]
[TFA][AT] Warface  TEC-9

**Comment by TheVogels** [comment]
There are more addons with this bug. Like a jetpack swep.

**Comment by TheVogels** [comment]
Also physics

**Comment by Mrleplayz** [comment]
yeah this is true for e2 played sounds aswell, apparently sometimes soundStop() does nothing, soundPurge() also sometimes does nothing by the looks of things
Experienced both of those while working
